common/math/big.go

```diff
@@ -78,7 +78,7 @@ func (i *HexOrDecimal256) MarshalText() ([]byte, error) {
 	return []byte(fmt.Sprintf("%#x", (*big.Int)(i))), nil
 }
 
-// Decimal256 unmarshals big.Int as a decimal string. When unmarshalling,
```

Comment:
This change is wrong and the line should be kept as-is.

Reply:
Thanks!
Reverted